spec/models/alchemy/page_spec.rb

```diff
@@ -189,28 +189,6 @@ module Alchemy
           end
         end
 
-        context "public has changed" do
-          it "should update published_at" do
-            expect {
-              page.update_attributes!(public: true)
-            }.to change { page.read_attribute(:published_at) }
-          end
-
-          it "should not update already set published_at" do
-            page.update_attributes!(published_at: 2.weeks.ago)
-            expect {
-              page.update_attributes!(public: true)
-            }.to_not change { page.read_attribute(:published_at) }
-          end
-        end
-
-        context "public has not changed" do
-          it "should not update published_at" do
-            page.update_attributes!(name: 'New Name')
-            expect(page.read_attribute(:published_at)).to be_nil
-          end
-        end
-
         context 'with elements already on the current page version' do
```

Comment:
Is there still a spec for updating of the cache key when you change the published version?